documentation/plain_rules.md

```diff
@@ -936,7 +946,6 @@ will propagate to the core, (and so making available to the EPL sentence) the fi
     "isPattern": "false",
     "subservice": "/",
     "service": "unknownt",
-    "position": "40.418889, -3.691944",
```

Comment:
Waiting for issue #579 fixing, so instead of delete this line it would be:

```
"location": "\"40.418889, -3.691944\"",
```

Is that one correct @AlvaroVega ?